src/containers/ConceptPage/components/ConceptMetaData.jsx

```diff
@@ -32,11 +43,23 @@ const ConceptMetaData = ({ t, licenses }) => (
         <FormikMetaImageSearch metaImageId={field.value} {...field} />
       )}
     </FormikField>
+    <FormikField name="subjects" label={t('form.subjects.label')} obligatory>
```

Comment:
Heller ikkje obligatorisk. I tillegg er form.subjects.label oversatt med Emne. Det er feil. Må være Fag.
For concept heng fag og nøkkelord sammen så dei burde også stå sammen. Kan flytte nøkkelord under fagvelger.